Mathlib/Data/Nat/Basic.lean

```diff
@@ -165,7 +165,6 @@ theorem eq_of_le_of_lt_succ {n m : ℕ} (h₁ : n ≤ m) (h₂ : m < n + 1) : m
 -- Moved to Std
 #align nat.one_add Nat.one_add
 
-@[simp]
```

Comment:
Is this `@[simp]` annotation useless / disallowed / redundant?

Reply:
I think this was demanded by the linter, so it's one of the three options.